TestLockAndUnlock fail

--- FAIL: TestLockAndUnlock (0.04s)
	lock_test.go:94: unexpected blocking
FAIL
coverage: 75.8% of statements
FAIL	github.com/coreos/etcd/pkg/fileutil	0.260s

--- a/pkg/fileutil/lock_test.go
+++ b/pkg/fileutil/lock_test.go
@@ -90,7 +90,7 @@ func TestLockAndUnlock(t *testing.T) {
 	// the previously blocked routine should be unblocked
 	select {
 	case <-locked:
-	case <-time.After(10 * time.Millisecond):
+	case <-time.After(20 * time.Millisecond):
 		t.Error("unexpected blocking")
 	}
 }

